docs/gambit/mutation-verifier.md

```diff
@@ -18,19 +18,21 @@ Installation instructions can be found [here](https://www.java.com/en/download/h
 
 ## Running the Mutation Verifier
 
-- Example:
+You will need to update your `certora-cli-alpha-master` installation using `pip` to get the relevant
+dependencies. Then you can run Gambit like so:
+
 ```
-java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/example.conf
+mutationTest path/to/config/file/Example.conf
 ```
 
 - **NOTE: if a path has spaces, put quotes around it to ensure correct parsing by Kotlin's libraries**
 
 ```{note}
-Gambit supports {ref}`--staging`.
+Gambit supports {ref}`--staging` and {ref}`--cloud`.
 
 However, Gambit currently has trouble with
-{ref}`--send_only` and {ref}`--cloud` in the run scripts.
-If you have these flags, please remove them for now.
+{ref}`--send_only` in the run scripts.
+If you have this flag, please remove it for now.
```

Comment:
I would move this entire note to the `"run_script"` section; there's not enough context here to understand what this means.